package.json

```
@@ -19,65 +19,65 @@
     "s3:start": "docker-compose up s3"
   },
   "dependencies": {
-    "@aws-sdk/client-s3": "^3.637.0",
+    "@aws-sdk/client-s3": "^3.679.0",
     "@lucia-auth/adapter-drizzle": "^1.1.0",
-    "@radix-ui/react-avatar": "^1.1.0",
-    "@radix-ui/react-dialog": "^1.1.1",
-    "@radix-ui/react-dropdown-menu": "^2.1.1",
+    "@radix-ui/react-avatar": "^1.1.1",
+    "@radix-ui/react-dialog": "^1.1.2",
+    "@radix-ui/react-dropdown-menu": "^2.1.2",
     "@radix-ui/react-label": "^2.1.0",
-    "@radix-ui/react-popover": "^1.1.1",
+    "@radix-ui/react-popover": "^1.1.2",
     "@radix-ui/react-scroll-area": "^1.2.0",
-    "@radix-ui/react-select": "^2.1.1",
+    "@radix-ui/react-select": "^2.1.2",
     "@radix-ui/react-separator": "^1.1.0",
     "@radix-ui/react-slot": "^1.1.0",
-    "@radix-ui/react-tooltip": "^1.1.1",
+    "@radix-ui/react-tooltip": "^1.1.3",
     "@t3-oss/env-nextjs": "^0.10.1",
     "@tanstack/react-form": "^0.34.1",
-    "@tanstack/react-query": "^5.53.1",
+    "@tanstack/react-query": "^5.59.16",
     "@tanstack/zod-form-adapter": "^0.34.1",
-    "@trpc/client": "^11.0.0-rc.490",
-    "@trpc/react-query": "^11.0.0-rc.490",
-    "@trpc/server": "^11.0.0-rc.490",
+    "@trpc/client": "^11.0.0-rc.599",
+    "@trpc/react-query": "^11.0.0-rc.599",
+    "@trpc/server": "^11.0.0-rc.599",
     "cmdk": "1.0.0",
-    "country-flag-icons": "^1.5.12",
+    "country-flag-icons": "^1.5.13",
     "cva": "^1.0.0-beta.1",
     "date-fns": "^4.1.0",
     "drizzle-orm": "^0.33.0",
-    "lucia": "^3.2.0",
+    "lucia": "^3.2.2",
     "lucide-react": "^0.396.0",
-    "next": "15.0.1",
+    "next": "^15.0.2-canary.9",
```

```suggestion
    "next": "^15.0.1",
```

Dont use canary version for next use the stable version. React uses release candidate because it is required in the stable Next15 version